### PR TITLE
WIP: Ecto 1.1 and Phoenix

### PR DESCRIPTION
**Pull request by ericmj**

### CODE REVIEW

File: lib/hex_web/api/key.ex [hunk 1]
```diff
@@ -33,7 +34,7 @@ defmodule HexWeb.API.Key do
       |> put_change(:secret_second, second)
 
     if changeset.valid? do
```
Comment (josevalim) [inline lib/hex_web/api/key.ex]: You want to use `HexWeb.Repo.insert` instead of checking `changeset.valid?`.

File: lib/hex_web/package.ex [hunk 2]
```diff
@@ -129,7 +129,7 @@ defmodule HexWeb.Package do
     changeset = changeset(package, :update, params)
 
     if changeset.valid? do
```
Comment (josevalim) [inline lib/hex_web/package.ex]: Same here.

**Comment by josevalim** [comment]
Looks great! Less code than I expected. :)

**Comment by pragmaticivan** [comment]
Will that be a good time if I clone this branch and start developing the template inside Phoenix?
Small preview: http://pragmaticivan.github.io/hex-web-template/
http://pragmaticivan.github.io/hex-web-template/packages.html

**Comment by ericmj** [comment]
This branch only contains changes to upgrade ecto to 1.1. I will merge this into master and create a separate branch for converting to phoenix.

**Comment by pragmaticivan** [comment]
Cool. Thanks.

**Comment by josevalim** [comment]
:shipit:

**Pull request merged**